cloudflare/resource_cloudflare_firewall_rule_test.go

```diff
@@ -84,3 +84,49 @@ func testFirewallRuleConfig(resourceID, zoneID, paused, description, expression,
 		}
 		`, resourceID, zoneID, paused, description, expression, action, priority)
 }
+
+func TestFirewallRuleBypass(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := "cloudflare_firewall_rule." + rnd
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+
+	filterQuoted := `(http.host eq \"` + rnd + `\")`
```

Comment:
this rule is invalid but i'm unsure why it's not failing when creating it in the tests (as the rule is parsed for validity on creation).

```suggestion
	filterQuoted := `(http.host eq \"terraform.cfapi.net\")`
```

(or pull it from the env using `CLOUDFLARE_DOMAIN` if you'd rather)

Reply:
I checked on CF and any string works.

But I've changed it to proposed `CLOUDFLARE_DOMAIN`,